test/stripe/invoice_test.rb

```diff
@@ -73,6 +73,26 @@ class InvoiceTest < Test::Unit::TestCase
         assert invoice.kind_of?(Stripe::Invoice)
       end
 
+      should "retrieve upcoming invoices with items" do
+        items = [{
+          plan: 'gold',
+          quantity: 2
+        }]
+
+        invoice = Stripe::Invoice.upcoming(
+          customer: "cus_123",
+          subscription_items: items
+        )
+
+        assert_requested :get, "#{Stripe.api_base}/v1/invoices/upcoming",
+          query: {
+            customer: "cus_123",
+            :'subscription_items[][plan]' => 'gold',
+            :'subscription_items[][quantity]' =>  2
+          }
```

Comment:
Yeah, we're still testing against some older Ruby versions that don't support that colon syntax on strings. Thanks.